.github/workflows/build-and-test.yml

```diff
@@ -213,6 +213,7 @@ jobs:
           git diff --exit-code '.github/ISSUE_TEMPLATE' || (echo 'Dropdowns in issue templates are out of date, please run "make generate-gh-issue-templates" and commit the changes in this PR.' && exit 1)
   unittest-matrix:
     strategy:
+      fail-fast: false
```

Comment:
nit: Why is this necessary?

Reply:
For testing it makes it easier to see all failures at once. It. is not required for this pr. Just a nice to have.